### PR TITLE
default content removed

### PR DESCRIPTION
**Pull request by claramiralles**
Hi @valerialp! I just removed the default content in App.js that is created when you start a react repo. Can you please double check and aprove the pull request? let me know if you need any further info!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.scss';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
```
Comment (valerialp) [inline src/App.js]: I think that this label can be removed

Comment (claramiralles) [reply src/App.js]: ready

Comment (claramiralles) [reply src/App.js]: Thanks @valerialp. I removed it. Please let me know if you approve the pulll request to proceed :)

Comment (valerialp) [reply src/App.js]: Good, you can merge.

Comment (claramiralles) [reply src/App.js]: great!

**Pull request merged**